src/admin/resolvers/mutations/ApprovedItem.ts

```diff
@@ -180,7 +187,8 @@ export async function rejectApprovedItem(
     url: approvedItem.url,
     title: approvedItem.title,
     topic: approvedItem.topic,
-    language: approvedItem.language,
+    // Make sure the language code is in upper case.
+    language: approvedItem.language.toUpperCase(),
```

Comment:
if this data is coming out of the db (and we're enforcing case going _in_ to the db), should we ever need to uppercase this value?

Reply:
Yeah that's what my previous comment also mentions. If we never allow to a non lowercase value be created in the first place, we should only have uppercase values

Reply:
Oh, that is my mistake. I thought I took out all of these out in a previous commit.